examples/webgpu_points_fat.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgpu - points - fat</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+
+	<body>
+
+		<div id="container"></div>
+
+		<div id="info"><a href="https://threejs.org" target="_blank">three.js</a> - fat points</div>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js",
+					"three/addons/": "./jsm/",
+					"three/nodes": "./jsm/nodes/Nodes.js"
+				}
+			}
+		</script>
+
+		<script type="module">
+
+			import * as THREE from 'three';
+
+			import WebGPU from 'three/addons/capabilities/WebGPU.js';
+			import WebGL from 'three/addons/capabilities/WebGL.js';
+
+			import WebGPURenderer from 'three/addons/renderers/webgpu/WebGPURenderer.js';
+
+			import Stats from 'three/addons/libs/stats.module.js';
+
+			import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+
+			import { FatPoints } from 'three/addons/points/FatPoints.js';
+			//import { FatPointsNodeMaterial } from 'three/addons/points/FatPointsNodeMaterial.js'; // why not this, instead?
```

Comment:
You can actually use just FatPointsMaterial and it would be auto-converted to FatPointsNodeMaterial by WebGPURenderer based on its name.

Reply:
Here I just meant that you can import just the FatPointsMaterial instead of FatPointsNodeMaterial and it should work exactly the same.
For background I believe setting `scene.background = THREE.Color( 0x222222 )` should also work just fine as `scene.backgroundNode = color( 0x222222 )`. So basically using the same code as for WebGL should work 🙂

Reply:
I think we no longer need `FatPointsMaterial`.  My intention is to remove it. (I just wanted to get its shader to work correctly before attempting the Node version.)

Reply:
>You can actually use just `FatPointsMaterial` and it would be auto-converted to `FatPointsNodeMaterial`

It does render, but the vertex colors are not honored.

Assuming that will be fixed, would it be preferable to retain `FatPointsMaterial` (in glsl) and remove `FatPointsNodeMaterial`?

ping @sunag @LeviPesin

Reply:
If the main scope is `WebGPURenderer`, I vote yes. 

It's great to see `NodeMaterial` saved lines of code in relation to the same material created in `ShaderMaterial`, with the advantages that each "uniform" is a channel for a procedural node.

Reply:
> Assuming that will be fixed, would it be preferable to retain FatPointsMaterial (in glsl) and remove FatPointsNodeMaterial?

No, I meant just that it autoconverts the type -- so just FatPointsMaterial could be imported. But having FatPointsNodeMaterial in node materials is still required 😅

Reply:
Yes, the target is `WebGPURenderer`.  I created this PR so the WebGPU backend would support points having width greater than 1 px.

@sunag So you vote "yes" on what? What should I delete from this PR?

Reply:
> @sunag So you vote "yes" on what? What should I delete from this PR?

Keep `FatPointsNodeMaterial` and remove `FatPointsMaterial` as your first suggestion. `Node Material` are more powerful and allow operate like `FatPointsMaterial` if necessary.

Reply:
>For background I believe setting scene.background = THREE.Color( 0x222222 ) should also work just fine as scene.backgroundNode = color( 0x222222 ).

For WebGPURenderer (both backends), it appears
```js
scene.background = new THREE.Color( 0xff0000 ); // does not honor viewport/scissor

scene.backgroundNode = color( 0xff0000 ); // honors viewport/scissor
```

Reply:
> scene.background = new THREE.Color( 0xff0000 ); // does not honor viewport/scissor

Could this be happening there because `scene.backgroundNode` overrides the `scene.background` settings?
I tested it in `webgpu_lines_fat` replacing all `scene.backgroundNode` to `null` and work.

Reply:
@sunag For me, the background color of the inset view extends to the entire screen when `scene.background` is used in WebGPURenderer.

Reply:
> > scene.background = new THREE.Color( 0xff0000 ); // does not honor viewport/scissor
> 
> Could this be happening there because `scene.backgroundNode` overrides the `scene.background` settings? I tested it in `webgpu_lines_fat` replacing all `scene.backgroundNode` to `null` and work. (or just remove all `scene.backgroundNode` assigns)

Setting the background color using the clear color doesn't work in the same way as WebGLRenderer.  The scissor area does not constrain the region cleared.

Reply:
/ping @sunag @LeviPesin See comment by @aardgoose .